.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,15 @@
+## Description
+<Provide a detailed explanation of the changes, why they were necessary, and their benefits or impact>
+<Explain the problem you are fixing or the feature you are adding>
+
+## Checklist
+- [ ] I have added test cases for this feature
+- [ ] I have updated the documentation (if necessary)
+- [ ] I have run and passed all tests
+- [ ] I have checked for code smells
+
+## Additional Information
+<Provide any additional information or context if necessary>
+
+## Related Issues or Pull Requests
+<Specify related issues or pull requests if any>
```

Comment:
I think we can move this section to the below of description, without title and with some example.

```suggestion
<Specify the issues this PR closes, by using closing words like `Closing #[number]`, `Fixes #[number]`>
```

Reply:
I think titles are good leading and organized.

Reply:
OK, let's try. 😊 

And what do you think about my other suggestions?

Reply:
Since we're removing "Description" title, I think we should either keep all titles or remove all.

Reply:
I think there is no problem about having some sub titles without starting with a title like description. For example: #547 

But I already suggested to remove this title since the information we'll add here will be only a few words (closing words). We can just mention them below the description (as we already do until now).